### ISSUE TITLE
[Classic] ESPN.com not working

### ISSUE THREAD

**Issue by strotee** [MrAlex94/Waterfox#1550]
I'm using WF 2020.04 and ESPN.com is not working. I'm presented with a gray screen and a "Terms of Use" section. I restarted WF with all add-ons disabled but I'm presented with the same issue. I've seen similar issues here and tried following those instructions to no avail, maybe I just didn't follow the right ones, I'm not sure. Is it a setting in about:config or has a recent change broke something? I don't recall seeing this in prior versions; the message on the site states their TOU changed on 4/14/2020.

https://i.ibb.co/JvKPsNQ/Untitled.png

Edit: I should mention that it works fine in Chromium and IE 11.

**Comment by hawkeye116477** [comment]
Is it on specific subpage or main page?

For me works good.

**Comment by strotee** [comment]
I just tried a page from their RSS feed and it returns the same blank/gray page. I do recall making changes to my system, give me a few minutes while I try a VM.

Edit: I'm not sure what I did to my system but it works fine in a 10 VM with only ublock origin installed.  Consider this a local issue and not a browser issue. Thanks

**Solved - Edit:** I solved it by changing Privacy/"Suppress HTTP Referrer Header"/changing the value to the Default value.
